recipes/tracktable/meta.yaml

```
@@ -0,0 +1,108 @@
+# https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html?highlight=meta.yaml
+
+{% set name = "tracktable" %}
+{% set version = "1.5.0" %}
+{% set build_number = 0 %}
+{% set sha256 = "afe9d27e4371ade061f195eaeb984468231245094ed0aaf294818f5e4b84c452" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # Building source from local dir
+  # path: ../..
+
+  # Building source from tarball or archive
+  url: https://github.com/sandialabs/tracktable/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }} # Generate hash command: curl -sL https://github.com/sandialabs/tracktable/archive/refs/tags/v{{ version }}.tar.gz | openssl sha256
+
+  # Building source from github repository
+  # git_url: https://github.com/sandialabs/tracktable.git
+  # git_rev: v{{ version }}
+  # git_tag: main # set the tag or branch to build from
+  # git_depth: 1 # Only pull the latest git changes, use -1 or comment out to get all git history
+
+  # Any patch files that should be built into the package
+  # patches:
+  #   - my.patch # the patch file is expected to be found in the recipe
+
+build:
+  number: {{ build_number }}
+  # string: debug # String for build version: 1.4.1 => 1.4.1-debug
+
+  # script_env: # Whitelist of env vars to pass to the build script
+    # - LD_LIBRARY_PATH # [linux]
+    # - DYLD_LIBRARY_PATH # [osx]
+    # - MYVAR
+    # - ANOTHER_VAR=some value
+
+requirements:
+
+  build:
+    # Conda-Build Dependencies
+    - {{ compiler('cxx') }}
+    # We need a recent CMake in order to support Boost 1.75.
+    - cmake >=3.19.5
+    - ninja  # [win]
+    - make   # [unix]
+    - setuptools
+
+  host:
+    # Python Dependencies
+    - python {{ python }}
```

No need for the variables, this will be handled on the feedstock with variant files auto-generated by conda-smithy. Same goes for the ones for `boost*`
```suggestion
    - python
```

---

numpy, python and boost packages no longer use the variant versions.